### PR TITLE
Dg 00006 namespace change

### PR DESCRIPTION
**Pull request by hadi-darwish**
## Description of the functionality:
replacing namespace with DigitalSplash
removing comments form unit test files

## Affected functionality
List of areas affected

## Testing procedures
Conducted tests and steps

## Related documentation changes
API documentation changes, etc

### CODE REVIEW

File: tests/PhpUnits/Helpers/HelperTest.php [hunk 1]
```diff
@@ -855,8 +854,8 @@ public function testEncryptLinkSuccess() {
 			);
 
 			$this->assertEquals(
-				str_replace("&", "[amp;]", base64_encode("https://rawadymario.com/projects?page=2&category=2")),
```
Comment (rawadymario) [inline tests/PhpUnits/Helpers/HelperTest.php:858]: When replacing a string in the whole project, it's better to replace using Case Sensitive. Please change the domain to be dg-splash

File: tests/PhpUnits/Helpers/MetaSeoTest.php [hunk 2]
```diff
@@ -58,15 +57,15 @@ public function testRenderFullSuccess() {
 				"PHP",
 			]);
 			MetaSeo::SetDescription("Mario Rawady is a Software Engineer");
-			MetaSeo::SetPhoto("https://rawadymario.com/assets/img/logo-big.png");
```
Comment (rawadymario) [inline tests/PhpUnits/Helpers/MetaSeoTest.php:61]: Here also

File: tests/PhpUnits/Helpers/ValidatorTest.php [hunk 3]
```diff
@@ -215,7 +214,7 @@ public function testValidateUsernameCharactersThrowError_01(): void {
 
 		public function testValidateUsernameSuccess(): void {
 			$this->assertTrue(
-				Validator::ValidateUsername("rawadymario")
```
Comment (rawadymario) [inline tests/PhpUnits/Helpers/ValidatorTest.php:218]: Here set the username to be: "digital_splash" or "digitalsplash" (Depending if we allow underscores in the username or not)

**Comment by hadi-darwish** [comment]
fixing

**Pull request closed without merging**